UPGRADING.md

```diff
@@ -74,6 +74,11 @@ allows an application to define handlers for these methods via `ExtendVoteHandle
 and `VerifyVoteExtensionHandler` respectively. Please see [here](https://docs.cosmos.network/v0.50/building-apps/vote-extensions)
 for more info.
 
+```diff
```

Comment:
Can we precise this is only needed when wiring manually an app (app v1). People using app v2 / depinject wont need to add that.

Reply:
Add a Upgrade section, but we also need `SetMigrationModuleManager` if do migration with app v2.